src/DCurves.jl

```diff
@@ -0,0 +1,68 @@
+
+"""
+    DCurves(M::Manifold, grid::AbstractVector)
+
+Space of curves on manifold `M` discretized on the given `grid`.
+"""
+struct DCurves{𝔽,TM<:Manifold{𝔽},TG<:AbstractVector} <:
+       Manifolds.AbstractPowerManifold{𝔽,TM,Manifolds.ArrayPowerRepresentation}
+    manifold::TM
+    grid::TG
```

Comment:
is this a vector of points, i.e. an irregular grid?

Reply:
So far I've only used uniform grids but here I allow irregular grid as well. It's supposed to be a vector of real numbers where the curve was discretized.

Reply:
I was thinking of irregular ones, since for regular ones just a set of stepsizes is necessary.